presto-orc/src/main/java/com/facebook/presto/orc/OrcDataSourceId.java

```diff
@@ -50,4 +50,30 @@ public String toString()
     {
         return id;
     }
+
+    /**
+     * Add the ORC file path to an exception to provide mode debug context.
```

Comment:
'debug mode'?

Reply:
No, the context as when one troubleshoots a failed query, and all they have is a stacktrace on the presto ui.